src/main/java/com/rmbcorp/javawriter/BuildJob.java

```diff
@@ -0,0 +1,25 @@
+package com.rmbcorp.javawriter;
+
+/**
+ * Created by Aspire on 10/29/2016.
+ */
+public interface BuildJob {
```

Comment:
this and JavacJob are quite similar.  review architecture and make changes if appropriate